contrib/build_executable.jl

```diff
@@ -109,37 +108,62 @@ function build_executable(exename, script_file, targetdir=nothing; force=false)
     run(`$(julia) $(build_sysimg) $(sys.buildfile) native $(userimgjl) --force`)
     println()
 
+    gcc = find_system_gcc()
+    # This argument is needed for the gcc, see issue #9973
+    win_arg = @windows ? "-D_WIN32_WINNT=0x0600" : ""
+    @windowsxp_only win_arg = "-D_WINVER=0x0502"
```

Comment:
Yes, try that, use it if it works. The important part isn't trying to build things on an XP machine, as much as making sure the results can run there.